streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -98,7 +99,7 @@ public SimpleBenchmark(File stateDir, String kafka) {
 
     public static void main(String[] args) throws Exception {
         String kafka = args.length > 0 ? args[0] : "localhost:9092";
-        String stateDirStr = args.length > 1 ? args[1] : "/tmp/kafka-streams-simple-benchmark";
+        String stateDirStr = args.length > 1 ? args[1] : TestUtils.tempDirectory("kafka-stream-system-test/kafka-streams-simple-benchmark").getAbsolutePath();
```

Comment:
Any reason why you didn't just use `TestUtils.tempDirectory()` ? i.e., does it need to have `kafka-streams-system-test/kafka-streams-simple-benchmark` ?